apps/server/src/utils/coerceType.ts

```diff
@@ -40,3 +58,35 @@ export function coerceNumber(value: unknown): number {
   }
   return parsedValue;
 }
+
+/**
+ * @description Converts a value to a colour if possible, throws otherwise
+ * @param {unknown} value - Value to be converted to a colour.
+ * @returns {string} - The converted value as a string.
+ * @throws {Error} Throws an error if the value is null or undefined.
+ */
+const hexColor = new RegExp(/^#((?:[a-f\d]{1,2}){3,4})$/, 'i');
+export function coerceColour(value: unknown): string {
+  if (value == null || typeof value != 'string') {
```

Comment:
I think this checks are exclusive, so something being of type string makes it not null or undefined.
In that case this could simply be
`if (typeof value !== "string")` <---- mind the !== (one more = operator)